src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -277,9 +279,15 @@
     }
 
     .manage-all-btn {
+        $manageAllBtnMargin: -5px 0 10px 0;
+
         float: right;
-        margin: -5px 0 10px 0;
+        margin: $manageAllBtnMargin;
         color: $bdl-box-blue;
+
+        &:hover {
```

Comment:
Can you make sure the :focus and :active states are ok here? The button CSS defines margins for those states, too.

Reply:
may be worth adding a mixin for this that lives with the button code, so we codify the selector that needs overriding instead of having to remember each time.

Reply:
Or a ButtonV2 😄

Reply:
Good call, :active needs the margin too. I also like the mixin idea